hazelcast-jet-core/src/main/java/com/hazelcast/jet/DAG.java

```diff
@@ -156,12 +156,14 @@ public DAG edge(Edge edge) {
         if (getInboundEdges(edge.getDestName())
                 .stream().anyMatch(e -> e.getDestOrdinal() == edge.getDestOrdinal())) {
             throw new IllegalArgumentException("Vertex '" + edge.getDestName()
-                    + "' already has an inbound edge at ordinal " + edge.getDestOrdinal());
+                    + "' already has an inbound edge at ordinal " + edge.getDestOrdinal()
+                    + ", use Edge.from().to() to specify an ordinal");
```

Comment:
This doesn't seem extra-helpful to me. The same message will be used regardless of the syntax used to create the edge. The user may have written `from(a, 1).to(b, 2)` and still get the message. Maybe it can be added only when we see that the edge is configured as 0-to-0?